controllers/users.controllers.js

```diff
@@ -0,0 +1,10 @@
+const { selectAllUsers } = require("../models/users.models");
+
+exports.getUsers = (req, res, next) => {
+  const { users } = req.query;
+  selectAllUsers()
+    .then((users) => {
```

Comment:
why are is req.query being used here in the controller? it's not used in the model anywhere with our db query and there's no guidance about using queries on this endpoint so let's delete it